### PR TITLE
makefile: RTLMP_FLOW is now more consistent, 1 if enabled and default empty

### PR DESCRIPTION
**Pull request by oharboe**
This is more consistent with e.g. GPL_TIMING_DRIVEN and other variables and should lead to fewer bugs

### CODE REVIEW

File: flow/Makefile [hunk 1]
```diff
@@ -657,7 +657,13 @@ endif
 
 # STEP 3: Timing Driven Mixed Sized Placement
 #-------------------------------------------------------------------------------
-ifeq ($(MACRO_PLACEMENT)$(MACRO_PLACEMENT_TCL)$(RTLMP_FLOW),)
+ifneq ($(MACRO_PLACEMENT)$(MACRO_PLACEMENT_TCL),)
+RUN_TDMS:=1
+endif
+ifeq ($(RTLMP_FLOW), 1)
+RUN_TDMS:=1
```
Comment (maliberty) [inline flow/Makefile]: This looks to be the opposite of the previous logic.

**Review by maliberty — APPROVED** [review]

**Comment by gudeh** [comment]
Hi guys @maliberty @oharboe @AcKoucher , was this PR supposed to change the behavior of the flow? because it seems that after this PR we now run TDMS (2_3) for designs that run MPL2.

**Comment by oharboe** [comment]
> Hi guys @maliberty @oharboe @AcKoucher , was this PR supposed to change the behavior of the flow? because it seems that after this PR we now run TDMS (2_3) for designs that run MPL2.

Oops. No.

**Comment by oharboe** [comment]
I can look at this tomorrow

**Pull request merged**